include/zephyr/posix/netinet/in.h

```diff
@@ -40,6 +40,8 @@ typedef uint32_t in_addr_t;
 && ((const struct in6_addr *)addr)->s6_addr32[1] == 0         \
 && ((const struct in6_addr *)addr)->s6_addr32[2] == 0xffff)   \
 
+#define INADDR_BROADCAST ((in_addr_t)0xffffffff)
```

Comment:
Already present in `include/zephyr/net/net_ip.h` by commit 606eeed8c92f

Reply:
Dropped.